arrebol/worker/auth/tolerant/tolerant.go

```diff
@@ -0,0 +1,22 @@
+package tolerant
+
+import (
+	"github.com/google/logger"
+	"github.com/ufcg-lsd/arrebol-pb/arrebol/worker/auth/token"
+)
+
+type Authorizer struct{}
+
+func NewAuthorizer() *Authorizer {
+	return &Authorizer{}
+}
+
+func (sa *Authorizer) Authorize(token *token.Token) error {
```

Comment:
in both cases, with or without allowlist, we must verify the token

Reply:
done.